tests/e2e/cairo_0_1_mixed/test_cairo_0_1_mixed_building.py

```diff
@@ -37,8 +41,8 @@ def test_cairo1_build_with_contract_names_build_together(
     assert build_path.exists()
     built_files = list(build_path.glob("*"))
     assert set(built_files) == set(
-        Path("build") / (contract + ".json") for contract in contracts
-    )
+        Path("build") / (contract + ".casm.json") for contract in contracts
+    ).union({Path("build") / (contract + ".sierra.json") for contract in contracts})
```

Comment:
I know this is a nit but please note that we go through the `contracts` collection twice which is inefficient. I think we can go ahead and use a standard loop instead of comprehensions.